CPP-Example.cpp

```diff
@@ -11,16 +11,17 @@ int main()
 
    strcpy(dest, src); 
  
-   strncpy(dest,src, strlen(dest)); //DevSkim: ignore DS111237,DS154189 
```

Comment:
[Banned C function detected (strncpy)](https://github.com/Microsoft/DevSkim/blob/master/guidance/DS111237.md#ruleDS111237)
strncpy is dangerous, as if the source contains 'n' or more characters, it will not null terminate the destination.

*Fix Guidance:* Use strcpy_s or strlcpy if possible. If no safe function is viable, strcpy/strncpy should be proceeded by conditional checks to verify tha that the source string will fit in the destination with a null termnator.


Suggestions:
**[1]** -   strcpy_s(dest,  strlen(dest, src)); //DevSkim: ignore DS111237,DS154189 
**[2]** -   strlcpy(dest, src,  strlen(dest)); //DevSkim: ignore DS111237,DS154189